trestle/core/commands/author/ssp.py

```diff
@@ -172,15 +172,20 @@ def _merge_imp_reqs(
         for imp_req in imp_reqs:
             if imp_req.control_id in control_map:
                 imp_req.uuid = control_map[imp_req.control_id].uuid
-            for statement in as_list(imp_req.statements):
+            statements: List[ossp.Statement] = as_list(imp_req.statements)
+            for statement in statements:
```

Comment:
This and all others can be simplified even further. I assume this is done as type hinting fails. If that is the case try to use this instead:

```
T = TypeVar('T')
def as_list(list_or_none: Optional[List[T]]) -> List[T]:
```

Reply:
type TG was already there so now using it. much better.